src/validators/json-schema.coffee

```diff
@@ -90,14 +127,25 @@ class JsonSchema
 
     results = []
 
-    #amanda to gavel converter
+    #TV4 to gavel converter
     if data.length > 0
       indexes = [0..data.length - 1]
       indexes.forEach (index) ->
         item = data[index]
-        pointer = item['property'] or []
+
+        if item['property'] == undefined
```

Comment:
This `if and the next `else if` can be written as `if not item['property]? then pathArray = []`